flexmeasures/utils/config_utils.py

```diff
@@ -81,7 +81,7 @@ def read_config(app: Flask, custom_path_to_config: str | None):
     path_to_config_home = str(Path.home().joinpath(".flexmeasures.cfg"))
     path_to_config_instance = os.path.join(app.instance_path, "flexmeasures.cfg")
 
-    # Custom config: not when testing (that should run completely on defaults)
+    # Custom config: not when testing (that should run completely on defaults) and documentation
```

Comment:
Either remove documentation from the comment as well, or re-add it as a test condition. I have a feeling it might be needed.